x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -189,7 +202,7 @@ class PackageClientImpl implements PackageClient {
     packageInfo: InstallablePackage,
     assetPaths: string[]
   ): Promise<InstalledAssetType[]> {
-    await this.#runPreflight();
+    await this.#runPreflight(INSTALL_PACKAGES_AUTHZ);
```

Comment:
added `installPackages` authz requirement here, though not sure if that's enough to reinstall transforms below